interview-questions.md

```diff
@@ -4,47 +4,47 @@ Answer the following questions. First, without external resources. Challenge you
 
 1. What is object destructuring?
 
-  Your answer:
+  Your answer: It is a method that takes in any number of state objects and allows them to be reffered to by shorthand for the duration of the current method you're in.
 
-  Researched answer:
+  Researched answer: After doing some research I discovered that destructing is a JavaScript feature that allows you to extract multiple pieces of data from an array or object and assign them to their own variables
 
 
 
 2. What is a DOM event?
 
-  Your answer:
+  Your answer: A DOM event occurs when code allows a user input to update the DOM
 
-  Researched answer:
+  Researched answer: // Had some trouble finding an answer that made sense to me //
 
 
 
 3. What is the difference between a class and an object?
 
-  Your answer:
+  Your answer: A class is a blueprint for which objects are created on
```

Comment:
A blueprint indeed! I think I'd consider the question a little more, as it's also asking about the differences between both class and an object. Your answer covers the class, but it kind of leaves object up in the air.